src/keychain.js

```diff
@@ -25,7 +25,8 @@ const defaultOptions = {
     iterationCount: 10000,
     salt: 'you should override this value with a crypto secure random number',
     hash: 'sha2-512'
-  }
+  },
+  passPhrase: 'correcthorsebatterystaple'
```

Comment:
What if the default password is just empty string?

Reply:
The code later [enforces that the password should be 20+ characters long](https://github.com/libp2p/js-libp2p-keychain/blob/f95fef4ad2e8bdf1d132c5264d8078bc5cf090f0/src/keychain.js#L112).